Code/Framework/AzCore/AzCore/Serialization/EditContext.h

```diff
@@ -235,6 +235,17 @@ namespace AZ
              */
             ClassBuilder*  ClassElement(Crc32 elementIdCrc, const char* description);
 
+
+             /**
+             * Declare element with attributes that belong to the class SerializeContext::Class, this is a logical structure, you can have one or more GroupElementToggles.
+             * T must be a boolean variable that will enable and disable each DataElement attached to this structure.
+             * \param description - Descriptive name of the field that will typically appear in a tooltip.
+             * \param memberVariable - reference to the member variable so we can bind to serialization data.
+             */
+            template<class T>
+            ClassBuilder* GroupElementToggle(const char* description, T memberVariable);
+
```

Comment:
At face value this seems like something that would be doable through the attributes instead of adding a new element. Are there any restrictions that prevent using attributes to mark groups?

Reply:
Well right now there is no current way to have a data element and a group in the same line, and this is the first part of a change that introduces new groupings in the editor: https://jira.agscollab.com/browse/ATOM-14004
Also, the enable/disable logic for the group makes it so that the user doesn't have to do a change notify for each element in the group explicitly.